www/js/controllers/home_controller.js

```diff
@@ -158,6 +158,32 @@
       return false;
     };
 
+    vm.IncidentsController.clearSearch = function() {
+      vm.IncidentsController.searchValue = '';
+    };
+
+    vm.IncidentsController.showClearSearchButton = function() {
+      if (vm.IncidentsController.searchValue) {
```

Comment:
Could be a one liner, like `return vm.IncidentsController.searchValue`. If you really want it to be true or false, it can be `return !!vm.IncidentsController.searchValue`